src/style.css

```diff
@@ -1,3 +1,74 @@
+:root {
+  --white: #fff;
+  --black: #000;
+  --blue: #00acdc;
+}
+
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  background-color: var(--white);
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.header-logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav-list {
+  margin: 0;
+  display: flex;
+  list-style: none;
+  padding: 0;
+  align-items: center;
+}
+
+.nav-link {
+  margin-right: 20px;
+  position: relative;
+}
+
+.nav-link a {
+  height: 60px;
+  display: flex;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+  font-size: 12px;
+  font-weight: 500;
+  color: var(--black);
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+a.is-active {
```

Comment:
- Don't use tag names for styling (except html and body)
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/7fb17028-374a-490f-852f-840cf297971e)